applications/FluidDynamicsApplication/python_scripts/navier_stokes_embedded_solver.py

```diff
@@ -8,6 +8,7 @@
 
 # Import applications
 import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+import KratosMultiphysics.MeshMovingApplication as KratosMeshMoving
```

Comment:
wouldn't it make more sense to import this only if you need it?
Otherwise this will be a restriction for users

Reply:
This check is already included in the Python solvers wrapper.

https://github.com/KratosMultiphysics/Kratos/blob/cd65b1a96bc72383416dabeb27dd93284f7cb52b/applications/FluidDynamicsApplication/python_scripts/python_solvers_wrapper_fluid.py#L68-L69

Reply:
but with what you have now you cannot use the embedded solver unless you have MehsMoving compiled
=> my point is not the check, but the unconditional import of MeshMoving
Or do you need it in any case?

Reply:
That's true... I only need it in case the FM-ALE is activated. Maybe is better to import it on the fly.

Reply:
yep
Take a look at the output of the tests...

Reply:
:man_facepalming: